databuilder/models/table_metadata.py

```diff
@@ -23,13 +24,40 @@ def __init__(self,
         self._name = name
         self._tag_type = tag_type
 
+    @staticmethod
+    def parse_tag_from_string(serialized_tag):
```

Comment:
are these two methods mainly for figuring out the badge from csv?

Reply:
yeah thats right. I could see how this should not belong to the model but to a csv extractor instead